gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/store/createFeed.ts

```diff
@@ -54,7 +54,7 @@ export default class CreateFeed extends SolanaCommand {
 
     const granularity = new BN(input.granularity)
     const liveLength = new BN(input.liveLength)
-    const length = new BN(8096)
+    const length = new BN(this.flags.length || 8096)
```

Comment:
Ability to configure length as requested by @archseer

Reply:
We should also validate that liveLength <= length

Reply:
yup! this is validated: https://github.com/smartcontractkit/chainlink-solana/pull/97/files#diff-ecea88c93003d6366639eac590fafe34bb4ee31afc3bc09f342ffeb528f79c67R64